go.mod

```diff
@@ -47,7 +52,7 @@ require (
 	github.com/sirupsen/logrus v1.9.3
 	github.com/skratchdot/open-golang v0.0.0-20200116055534-eef842397966
 	github.com/soheilhy/cmux v0.1.5
-	github.com/spf13/cobra v1.7.0
+	github.com/spf13/cobra v1.8.0
```

Comment:
Is this actually needed for the PR? Ideally I'd like to keep the package upgrades separate from the feature PRs

Reply:
Not sure how this one got pulled it, I'll change it back. Shouldn't be needed.

Reply:
I investigated this one some more when pulling in the new version of `argoprog/pkg`. It seems that the new version is a transitive dep of it, so 1.8.x is needed.